active/Win32/Window_and_Application_API_Spec.md

```diff
@@ -0,0 +1,655 @@
+Gets the 
+[LaunchActivatedEventArgs](https://docs.microsoft.com/uwp/api/Windows.ApplicationModel.Activation.LaunchActivatedEventArgs) 
+in a UWP app. This property is null in a Desktop app.
+
+```CS
+public Windows.ApplicationModel.Activation.LaunchActivatedEventArgs LaunchActivatedEventArgs { get; };
+```
+
+# API Details
+
+## Window class
+Microsoft.UI.Xaml namespace
+
+> The Window class' APIs in WinUI3 (**Microsoft**.UI.Xaml) match the existing system Window
+API (**Windows**.UI.Xaml), except for the following differences.
+
+```cs
+[webhosthidden]
+[contentproperty("Content")]
+unsealed runtimeclass Window
+{
+
+    ...
+
+    // These APIs match the system Xaml versions, except the event args are now from the Xaml namespace
+    event Windows.Foundation.TypedEventHandler<Object,Microsoft.UI.Xaml.XamlWindowActivatedEventArgs> Activated;
+    event Windows.Foundation.TypedEventHandler<Object,Microsoft.UI.Xaml.XamlWindowClosedEventArgs> Closed;
+    event Windows.Foundation.TypedEventHandler<Object,Microsoft.UI.Xaml.XamlWindowSizeChangedEventArgs> SizeChanged;
+    event Windows.Foundation.TypedEventHandler<Object,Microsoft.UI.Xaml.XamlWindowVisibilityChangedEventArgs> VisibilityChanged;
+
+    // New APIs
+    String Title;
+    Boolean ExtendsContentIntoTitleBar;
+    Microsoft.System.DispatcherQueue DispatcherQueue{ get; };
+}
+```
+
+## Application class
+
+Microsoft.UI.Xaml namespace
+
+> The Application class' APIs in WinUI3 (**Microsoft**.UI.Xaml) match the existing system Application
+API (**Windows**.UI.Xaml), except the OnLaunched virtual args is
+is now a Xaml type rather than Windows.ApplicationModel.Activation
+
+```cs
+[webhosthidden]
+unsealed runtimeclass Application
+{
+    ...
+    overridable void OnLaunched(Microsoft.UI.Xaml.XamlLaunchActivatedEventArgs args);
+};
+```
+
+## IWindowNative (COM) interface
+
+```cs
+[
+    object,
+    uuid( EECDBF0E-BAE9-4CB6-A68E-9598E1CB57BB ),
+    local,
+    pointer_default(unique)
+]
+interface IWindowNative: IUnknown
+{
+    [propget] HRESULT WindowHandle([out, retval] HWND* hWnd);
+};
+```
+
+## Event args
+
+```cs
+[webhosthidden]
+runtimeclass XamlLaunchActivatedEventArgs
+{
+    String Arguments{ get; };
+    Windows.ApplicationModel.Activation.LaunchActivatedEventArgs UWPLaunchActivatedEventArgs{ get; };
+};
+```
+
+```cs
+[webhosthidden]
+runtimeclass XamlWindowActivatedEventArgs
+{
+    Boolean Handled;
+    Microsoft.UI.Xaml.WindowActivationState WindowActivationState{ get; };
+};
+```
+
+
+```cs
+[webhosthidden]
+runtimeclass XamlWindowCreatedEventArgs
+{
+    Microsoft.UI.Xaml.Window Window{ get; };
+};
+```
+
+
+```cs
+[webhosthidden]
+runtimeclass XamlWindowSizeChangedEventArgs
+{
+    Boolean Handled;
+    Windows.Foundation.Size Size{ get; };
+};
+```
+
+```cs
+[webhosthidden]
+runtimeclass XamlWindowVisibilityChangedEventArgs
+{
+    Boolean Handled;
+    Boolean Visible{ get; };
+};
+```
+
+```cs
+[webhosthidden]
+runtimeclass XamlWindowClosedEventArgs
```

Comment:
Missing from spec. Did we intend to keep this? Or did we forget to delete it from the IDL?